instrumentation/opentelemetry-instrumentation-urllib/tests/test_urllib_integration.py

```diff
@@ -188,6 +189,18 @@ def test_suppress_instrumentation(self):
 
         self.assert_span(num_spans=0)
 
+    def test_suppress_http_instrumentation(self):
```

Comment:
Is there a test that tests the scenario of being instrumented with multiple http instrumentations?